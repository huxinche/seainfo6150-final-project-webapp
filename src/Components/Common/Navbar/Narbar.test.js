import React from "react";
import logo from "../../../images/UniversityLogo.png"
import NavBar from "./Navbar";
it("renders the NavBar component correctly", () => {
    const { container } = render(<NavBar logo={logo}/>);
    expect(container).toMatchSnapshot();
})