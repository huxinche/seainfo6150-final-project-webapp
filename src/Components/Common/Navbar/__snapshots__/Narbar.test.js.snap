// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the NavBar component correctly 1`] = `
<div>
  <div
    class="header"
  >
    <div
      class="title"
    >
      <a
        aria-current="page"
        class="active"
        href="/"
      >
        <b>
          B University
        </b>
         Graduate Education
      </a>
    </div>
    <div
      class="logo"
    >
      <img
        alt=""
        src="UniversityLogo.png"
      />
    </div>
  </div>
  <nav>
    <span
      class="toggle"
    >
      <i
        class="fas fa-bars"
      />
    </span>
    <ul
      class="nav hide"
    >
      <li>
        <a
          aria-current="page"
          class="nav-item selected"
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          class="nav-item"
          href="/findmyprogram"
        >
          Find Your Program
        </a>
      </li>
      <li>
        <a
          class="nav-item"
          href="/whybu"
        >
          Why BU
        </a>
      </li>
      <li>
        <a
          class="nav-item"
          href="/contactus"
        >
          Contact US
        </a>
      </li>
    </ul>
    <ul
      class="auth hide"
    >
      <li>
        <a
          class="nav-item"
          href="/register"
        >
          SignUp/Register
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
